tests/tt_metal/tt_metal/unit_tests_fast_dispatch_single_chip_multi_queue/command_queue/test_EnqueueWaitForEvent.cpp

```diff
@@ -247,6 +247,7 @@ TEST_F(MultiCommandQueueSingleDeviceFixture, TestEventsReadWriteWithWaitForEvent
 // ordered via events. Do many loops, occasionally increasing size of buffers (page size, num pages).
 // Ensure read back data is correct, data is different for each write.
 TEST_F(MultiCommandQueueSingleDeviceFixture, TestEventsReadWriteWithWaitForEventCrossCQs) {
+    GTEST_SKIP();
```

Comment:
can you prefix the test name with `DISABLED_` instead of `GTEST_SKIP()` and leave a comment to the issue you created?

Reply:
Instead of skipping this test for all Arch's, we disable this test only for GS Arch as according to @kmabeeTT's findings listed in. https://github.com/tenstorrent-metal/tt-metal/issues/6281

This ND issue does not appear to show in WH